dev-scripts/webterminal-localhost.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+#
+# Simple bash script to ease testing OpenShift Terminal changes by running bridge locally.
+#
+
+set -e
+
+function parseArgs() {
+  while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --setup) DO_SETUP="true";;
+        --frontend-patch) DO_FRONTEND_PATCH="true";;
+        --frontend) DO_FRONTEND="true";;
+        --backend) DO_BACKEND="true";;
+        --run) DO_SETUP="true"; DO_RUN="true";;
+        --all) DO_SETUP="true"; DO_FRONTEND="true"; DO_BACKEND="true"; DO_RUN="true";;
+        -h|--help) help; exit 0 ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+  done
+}
+
+function help() {
+  echo "This scripts helps to get web terminal working with your local changes available on localhost
+    Available args:
+      --setup: emulates in-cluster configuration which is needed for backend
+      --backend: compiles patched backend to unblock localhost
+      --frontend: compiles patched frontend to unblock localhost
+      --run: runs bridge (includes setup)
+      --all: includes all from above
+      
+      --frontend-patch: patches frontend to unblock you to compile and run manually
+      
+      --install: propagating this script into PATH. IS NOT IMPLEMENTED YET
+
+      --help: get this info message
+
+      Usage: 
+      Just make changes and do to get them complied and available at localhost:9000
+      ./webterminal-localhost.sh --all
+      "
+}
+
+# Set up
+function setup() {
+  echo "The following folder with link are going to be created
+    after you give sudo"
```

Comment:
Maybe something like 
```suggestion
  echo "Require sudo access to create /var/run/secrets/kubernetes.io/serviceaccount/ directory
     and create symlink to `/tmp/kube-in-cluster-emulation`"
```

Reply:
We should also maybe have an "--uninstall" or "--un-setup" argument to clean up dirs/symlinks we create.

Reply:
> We should also maybe have an "--uninstall" or "--un-setup" argument to clean up dirs/symlinks we create.

+100500 I thought about it but did not add even TODO )